core/src/main/java/dk/alexandra/fresco/framework/network/Network.java

```diff
@@ -1,7 +1,16 @@
 package dk.alexandra.fresco.framework.network;
 
+import java.net.ServerSocket;
 import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import dk.alexandra.fresco.framework.Party;
+import dk.alexandra.fresco.framework.configuration.NetworkConfiguration;
+import dk.alexandra.fresco.framework.configuration.NetworkConfigurationImpl;
+import dk.alexandra.fresco.framework.util.ExceptionConverter;
```

Comment:
Please clean up the imports that we no longer need. (Also, the imports should be in a single block. You may need to adjust the formatting in the editor you are using :)).

Reply:
I have fixed it. I have hopefully also fixed my editor now